### DISCUSSION TITLE
Some functional objectives

### DISCUSSION THREAD

**Discussion by Jbla0712** [ptrumpis/snap-camera-server#119] [Share]
Some functional objectives with fallback or blendmode corrected. 🎨
Have fun 🕺

**_**EDIT: My old lenses imported with the tool, before Snapchat banned their recovery.**
All lenses work but BE CAREFUL you must add the tables in ADMINER_**

( Lens mainly for women )

Update

Mainly fixed with blend "blendmode">16< with 13 value in the files "scene_scn.xml".
Sometimes the fallback disable.

![lens](https://github.com/user-attachments/assets/73ff22b4-ba47-4ee1-a575-3cce1f8928ff)

👉 https://github.com/Jbla0712/import.git

**Comment by Jbla0712** [comment]


**Reply by dcordova20** [reply]
how do i get them to work on my server?

**Reply by Jbla0712** [reply]
I made a more or less clear tutorial
https://youtu.be/Pp6ljLzRjnU?si=9uk7DCv_GbI9r10W
It can help you
You need to put in "import" folder in the snap camera server folder
